.github/workflows/build-images-from-branch.yml

```diff
@@ -41,7 +41,7 @@ jobs:
       - name: Get list of services
         id: get-services
         run: |
-          services=$(grep "^  [^ ]" docker-compose.yml | grep -v '#' | awk -F: '{print $1}' | sed -e 's/^ *//')
```

Comment:
This is probably not what we want as this looks for files with name "docker-compose.yml"

Reply:
Thanks, good catch! updating this